src/server/controllers/payment.controller.js

```diff
@@ -129,6 +129,49 @@ export const makePayment = async (req, res) => {
   }
 };
 
+export const makeGroupPayment = async (req, res) => {
+  const paymentCode = req.params.payment_code;
+  try {
+    const penaltyType = req.params.type;
+    const { slipNumber } = req.body;
+    const penaltyGroup = await penaltyGroupService.getByPaymentCode(paymentCode);
+    const penaltiesOfType = penaltyGroup.penaltyDetails.find(p => p.type === penaltyType).penalties;
+    const amountPaidForType = penaltyGroup.penaltyGroupDetails.splitAmounts
+      .find(s => s.type === penaltyType).amount;
+    const redirectUrl = `https://${req.get('host')}${config.urlRoot}/payment-code/${paymentCode}/${penaltyType}/receipt`;
+
+    const cpmsResp = await cpmsService.createGroupCashTransaction(
+      paymentCode,
+      penaltyGroup.penaltyGroupDetails,
+      penaltyType,
+      penaltyGroup.penaltyDetails,
+      slipNumber,
+      redirectUrl,
+    );
+
+    if (cpmsResp.data.code !== '000') {
```

Comment:
how do we know this is what we expect?

Reply:
I have observed in the logs a response with this code and the message `Success` - does your wiki doc cover all possible codes?